media/livestream/livestream_test.go

```diff
@@ -28,13 +28,16 @@ const (
 	deleteChannelEventResponse = "Deleted channel event"
 	deleteChannelResponse      = "Deleted channel"
 	deleteInputResponse        = "Deleted input"
+	deleteAssetResponse        = "Deleted asset"
 	startChannelResponse       = "Started channel"
 	stopChannelResponse        = "Stopped channel"
 	location                   = "us-central1"
 	inputID                    = "my-go-test-input"
 	backupInputID              = "my-go-test-backup-input"
 	channelID                  = "my-go-test-channel"
 	eventID                    = "my-go-test-channel-event"
+	assetID                    = "my-go-test-asset"
```

Comment:
please consider using unique identifiers, to avoid resource contention between multiple concurrent test runs. This could be:

```
var assetID = fmt.Sprintf("my-go-test-asset-%s", uuid.New().String())
```

there's no need to change the existing patterns, but it will produce better outcomes, and it means there's no need to delete the existing objects *before* testing.

(except for pool, which you have documented only allows one per location. Thanks for that!)

Reply:
Done